testing/simulator/src/basic_sim.rs

```diff
@@ -16,7 +16,7 @@ use tokio::time::sleep;
 use types::{Epoch, EthSpec, MinimalEthSpec};
 
 const END_EPOCH: u64 = 16;
-const GENESIS_DELAY: u64 = 32;
+const GENESIS_DELAY: u64 = 60;
```

Comment:
We might be able to revert this now we use prod `c-kzg-4844` and `rust-eth-kzg` for PeerDAS, which only takes 500ms to initialize.

Reply:
Done